workers/unity/Packages/io.improbable.gdk.loadbalancing/LoadBalancingHelper.cs

```diff
@@ -0,0 +1,77 @@
+using System;
+using System.Collections.Generic;
+using Improbable.Gdk.Core;
+using Unity.Entities;
+
+namespace Improbable.Gdk.LoadBalancing
+{
+    public static class LoadBalancingHelper
+    {
+        public static void AddLoadBalancingSystems(this WorkerInWorld worker, Action<LoadBalancerConfiguration> configure)
```

Comment:
What do you mean? 

```
LoadBalancerConfiguration AddLoadBalancingSystems(this WorkerInWorld worker);

or

void AddLoadBalancingSystems(this WorkerInWorld worker, Func<LoadBalancerConfiguration>);
```

Reply:
I mean `LoadBalancerConfiguration AddLoadBalancingSystems(this WorkerInWorld worker);` This will avoid additional allocations for captures in the Action

Reply:
So the reason I structured it this way is to scope the configuration of the load balancing, just made more sense to me at the time.

fwiw I don't think the `Action<>` allocation really matters at all, this isn't hot path by any means

Reply:
Discussed offline - going to leave this as is